log/noop/noop.go

```diff
@@ -0,0 +1,56 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package noop provides an implementation of the OpenTelemetry Logs Bridge API
```

Comment:
(this is not a blocker, feel free to resolve if you have different opinon)

How about

```suggestion
// Package noop provides an implementation of the [OpenTelemetry Logs Bridge API]
```

(and doing the same in other places)

And adding at the end:

```go
// [OpenTelemetry Logs Bridge API]: https://pkg.go.dev/go.opentelemetry.io/otel/log
```

Reply:
I didn't want to link to an unreleased package. I'll explicitly add this to the existing documentation issues and link after the release.